pkg/controller/installation/core_controller.go

```diff
@@ -783,7 +814,6 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	// traffic within the K8s cluster exists and has valid FQDN manager service
 	// names and localhost.
 	if instance.Spec.Variant == operator.TigeraSecureEnterprise && managementCluster != nil {
-		log.Info("Creating secret for internal manager credentials")
```

Comment:
Is this log removal intended?

Reply:
Yeah we now log when we create certs with `https://github.com/tigera/operator/blob/f8d001da43ac6701a8e1f076c0a1a0e5336dd097/pkg/controller/utils/certs.go#L62`

Reply:
I asked for that, it shows up on every reconciliation and when it's not actually creating anything.